modules/search/class.jetpack-search.php

```diff
@@ -429,6 +429,55 @@ public function search( array $es_args ) {
 		return $response;
 	}
 
+	/**
+	 * Displays HTML about the result of the spelling auto-correction.
+	 * Should be called during loop_start to be displayed below page headers and above post results.
+	 *
+	 * @since 6.4.2
+	 *
+	 * @param string $spell_text The text to show in the suggest block
+	 */
+	public function display_suggest_result( $spell_text ) {
+		printf("<div class=\"suggest-result page-header\"><h4>" . $spell_text . "<br><br></h4></div>");
+	}
+
+	/**
+	 * Displays HTML for "Did you mean, $suggestion?", which $suggestion being a link to search for the suggestion.
+	 * Should be called during loop_start to be displayed below page headers and above post results.
+	 *
+	 * @since 6.4.2
+	 *
+	 * @param WP_Query $query             The WP_Query object.
+	 */
+	public function filter__show_did_you_mean( $query ) {
+		if ( ! is_array( $this->search_result ) ) return;
```

Comment:
Nitpick: this (and similar) should have braces and be on multiple lines: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style